src/runtime/ranger/ranger_resource_policy_manager.cpp

```diff
@@ -226,9 +229,12 @@ dsn::error_code ranger_resource_policy_manager::update_policies_from_ranger_serv
         // for the newly created table, its app envs must be empty. This needs to be executed
         // periodically to update the table's app envs, regardless of whether the Ranger policy is
         // updated or not.
-        CHECK_EQ_MSG(dsn::ERR_OK, sync_policies_to_app_envs(), "Sync policies to app envs failed.");
-        LOG_DEBUG("Sync policies to app envs succeeded.");
-        return dsn::ERR_OK;
+        err_code = sync_policies_to_app_envs();
+        if (err_code == dsn::ERR_OK) {
+            LOG_DEBUG("Sync policies to app envs succeeded.");
```

Comment:
If move this log into `sync_policies_to_app_envs()`, you can simply call `ERR_LOG_AND_RETURN_NOT_OK` directly.